scripts/go2.lic

```diff
@@ -900,7 +954,7 @@ else
       while first < room_list.length
          respond
          for which in (first)..([(first+show_size-1),room_list.length].min)
-            respond "#{(which).to_s.rjust(5)}: #{room_list[which-1].title.first.ljust(37)} (#{room_list[which-1].id})"
+            respond "#{(which).to_s.rjust(5)}: #{room_list[which-1].title.first.ljust(37)} (#{room_list[which-1].id}) #{room_list[which-1].location.to_s.gsub(/(^ *the *)?(town|hamlet|city|village|plains|free port|lowlands|tunnels and caverns|foothills|holdings|southern part|northern slopes|northern reaches|environs|somewhere)?( o[fn] )?|,.*$/i,"")}  %s" % [ room_list[which-1].uid ]
```

Comment:
How will this affect DragonRealms and lack of UID?

Reply:
map_dr.rb defines uid for rooms, but they're just empty.  It won't cause errors but is unnecessary for them until they get access to uids.  (further conversation was had and mrhoribu is in the process of making corrective edits by game).